frontend/src/metabase/search/components/UserNameDisplay/UserNameDisplay.tsx

```diff
@@ -0,0 +1,37 @@
+import { t } from "ttag";
+import { useUserListQuery } from "metabase/common/hooks/use-user-list-query";
+import { Text } from "metabase/ui";
+import type { UserId } from "metabase-types/api";
+
+export type UserNameDisplayProps = {
+  value: UserId | null;
+  title: string;
```

Comment:
Nit:
- `title` suggests https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/title
- `label` or `placeholder` feels more appropriate
- this comment applies to `CreatedByFilter.title` as well (`placeholder` does not work in that context though)